trappy/plotter/AbstractDataPlotter.py

```diff
@@ -73,16 +74,22 @@ def _parse_value(self, signal_def):
         :type signal_def: str
         """
 
-        event, column = signal_def.split(":")
+        match = re.match("(?P<event>[^:]+):(?P<column>[^:]+)(:.+)?", signal_def)
```

Comment:
And what about this simple mod...
```
color = [int(n) if 'x' not in n else int(n, 16) for n in color[1:].split(",")]
```

That should allow to specify colors in HEX format as well, e.g.
```
color = ':0,0xFF,12'
```